src/Host/Configurations/Startup.cs

```diff
@@ -1,13 +1,15 @@
 namespace DN.WebApi.Host.Configurations;
-
+using Serilog;
 internal static class Startup
 {
+    private static readonly ILogger _logger = Log.ForContext(typeof(Startup));
     internal static ConfigureHostBuilder AddConfigurations(this ConfigureHostBuilder host)
     {
         host.ConfigureAppConfiguration((context, config) =>
         {
             const string configurationsDirectory = "Configurations";
             var env = context.HostingEnvironment;
+            _logger.Information($"Enviroment Mode: {env.EnvironmentName}");
```

Comment:
Environment name is already logged at application startup:

![image](https://user-images.githubusercontent.com/775707/147615640-268ef3d5-89fc-4a46-8ce1-5da6e3bec1a2.png)

Reply:
This ended up as a misprint , sorry.